Models/SaveImage.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SaveImage.swift
+//  iosdevapp
+//
+//  Created by Leonit Ejupi on 02.03.23.
+//
+
+import Foundation
+import UIKit
+
+extension GenerateImageViewModel {
```

Comment:
I would create a standalone service for the saving image functionality instead of having a view model extension.